lib/routes/admin-api/export.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const { ADMIN } = require('../../permissions');
+const Controller = require('../controller');
+const yaml = require('js-yaml');
+const { handleErrors } = require('./util');
+
+class ExportController extends Controller {
+    constructor(config) {
+        super(config);
+        this.get('/', this.export, ADMIN);
+    }
+
+    async export(req, res) {
+        const { strategies, features, format } = req.query;
+
+        try {
+            const data = await this.config.importExport.export({
+                exportStrategies: strategies !== '0',
+                exportFeatures: features !== '0 ',
+            });
+            if (format === 'yaml') {
+                res.type('yaml')
+                    .send(yaml.safeDump(data))
+                    .end();
+            } else {
+                res.json(data).end();
+            }
```

Comment:
As this is intended for the UI I think it would make sense to return a file/attachment.